### DISCUSSION TITLE
is there a continuous jog function in the Pendant app?

### DISCUSSION THREAD

**Discussion by bLouChip** [winder/Universal-G-Code-Sender#2458] [Help] [answered]
Specifically in the Pendant mobile app, is there a way or a key to invoke continuous jog, such as when holding the arrow softkey in the Windows UGS program ?

**Comment by breiler** [comment, marked as answer]
No not yet. I have just prepared the web server to support websockets which is required for these kinds of features. 

Unfortunately most of the web frameworks that we are using are outdated and needs upgrading/replacing before this can be implemented.
